layouts/partials/resource-link.html

```diff
@@ -38,7 +38,7 @@ <h3 class="mt-4 sm:text-medium text-xl font-bold text-gray-700 sm:col-span-2 sm:
     {{ end }}
   </div>
 
-  <div class="flex flex-wrap gap-2 text-xs lowercase">
+  <!-- <div class="flex flex-wrap gap-2 text-xs lowercase">
```

Comment:
I would be inclined to just remove the whole section at present instead of leaving it commented out. Either way works of course.